toolset/utils/output_helper.py

```diff
@@ -8,80 +10,50 @@
 FNULL = open(os.devnull, 'w')
 
 
-def header(message,
-           top='-',
-           bottom='-',
-           log_file=None,
-           quiet=False,
-           color=None):
-    '''
-    Generates a clean header with the given message and top/bottom text breaks.
-    Optionally, a log file can be provided to log this header to, as well.
-    '''
-    topheader = (top * 80)[:80]
-    bottomheader = (bottom * 80)[:80]
-    result = ""
-    if topheader != "":
-        if color is not None:
-            result += color
-        result += "%s" % topheader
-    if message != "":
-        if color is not None:
-            result += color
-
-        if result == "":
-            result = "  %s" % message
-        else:
-            result += "%s  %s" % (os.linesep, message)
-    if bottomheader != "":
-        if color is not None:
-            result += color
-        result += "%s%s" % (os.linesep, bottomheader)
-    log(result + os.linesep, None, log_file, True, quiet)
-
-
-def log(log_text=None,
-        prefix=None,
-        log_file=None,
-        allow_newlines=False,
-        quiet=False):
+def log(log_text=None, **kwargs):
```

Comment:
Please no `kwargs`. If you're just going to call them out later (`kwargs.get('color','')` for example), then just specify them here.